packages/genesys-spark-components/src/components/beta/gux-link/example.html

```diff
@@ -0,0 +1,35 @@
+<h2>Standalone Link</h2>
+<gux-link-beta standalone>
+  <a href="#" target="_top">Link</a>
+</gux-link-beta>
```

Comment:
From the ticket it mentions that a standalone link always has an icon at the end or has this changed ? I did read that the breadcrumbs do use the the `standalone` without the icon at the end but just wanted to confirm.

Reply:
It does not necessarily, but highly recommended iirc. I clarified it with UX. They were fine with the solution of slotting an icon. It was also the best option I could think of with edge cases like breadcrumbs.

I guess it would make sense to remove the non-icon example though